src/OpenLoco/Windows/LandscapeGeneration.cpp

```diff
@@ -1049,7 +1049,7 @@ namespace OpenLoco::Ui::Windows::LandscapeGeneration
 
                 case widx::number_of_towns_down:
                 {
-                    uint16_t newNumTowns = std::max<uint16_t>(0, options.numberOfTowns - 1);
+                    uint16_t newNumTowns = std::max<uint16_t>(1, options.numberOfTowns - 1);
```

Comment:
Could perhaps make this a constant defined in the town manager

Reply:
`kMaxTowns` is defined in `S5::Limits`, so it makes sense to me to put `kMinTowns` in the same spot, what do you think?

Reply:
sure